emr/launch_emr.py

```diff
@@ -176,12 +247,16 @@ def launchemr(name=None):
 
     tunnel.stop()
 
-    print("Copying notebooks back to local machine")
-    shutil.rmtree(os.path.join(here, "code"))
-    with pysftp.Connection(
-        ip_address, username="hadoop", private_key=mypkey, cnopts=cnopts
-    ) as sftp:
-        sftp.get_r("code", here)
+    print(f"Copying notebooks back to local machine: {local_notebooks_dir}...")
+    with tempfile.TemporaryDirectory() as tmpdir:
+        with pysftp.Connection(
+            master_address, username="hadoop", private_key=mypkey, cnopts=cnopts
+        ) as sftp:
+            sftp.get_r("dsgrid_notebooks", tmpdir)
+        shutil.rmtree(local_notebooks_dir)
+        shutil.copytree(
+            os.path.join(tmpdir, "dsgrid_notebooks"), str(local_notebooks_dir)
+        )
```

Comment:
What if a user wants to work in a different folder? This is not a simple command they can replicate on the command line.